charts/istio-alerts/values.yaml

```diff
@@ -117,3 +118,17 @@ serviceRules:
 
     # -- How long to evaluate the latency of services.
     for: 15m
+
+  # -- Does a basic lookup using the defined selectors to see if we can see any
+  # info for a given selector. This is the "watcher for the watcher". If we get
+  # alerted by this, we likely have a bad selector and our alerts are not going
+  # to ever fire.
+  destinationServiceSelectorValidity:
+    # -- Whether to enable the monitor on the selector for the VirtualService.
+    enabled: true
+
+    # -- Severity of the monitor
+    severity: warning
```

Comment:
Are we going to set this to critical later - or leave it as warning? if we're leaving it as warning (and it feels important), let's maybe describe our thinking here?

Reply:
This was a pattern match to the other alerts. I don't know when/why we are allowing the severity to be set. My gut says that most users are probably not overriding this.

Reply:
So the question to you is - is this a critical alert, or do we consider this a warning? We want the default to be "right" here for most cases. What is your gut?

Reply:
So this is a warning because it is not saying your actual inspected thing is actively breaking. it is just saying you have loss of observability. Depending on the service this could be configured to be a critical problem if you prefer to be really on top of monitoring. This is not a reality for all teams/end-users tho due to prioritization. So I think default to warning and let it be configured for critical is the right path here.